scipy/stats/boost/tests/test_boost_stats.py

```diff
@@ -0,0 +1,77 @@
+'''Boost stats tests.'''
+
+import numpy as np
+from scipy.stats import (
+    bernoulli as scipy_bernoulli,
+    beta as scipy_beta,
+    nbinom as scipy_nbinom,
+    binom as scipy_binom,
+    ncx2 as scipy_ncx2,
+)
+from scipy.stats.boost import (
+    bernoulli as boost_bernoulli,
+    beta as boost_beta,
+    nbinom as boost_nbinom,
+    binom as boost_binom,
+    ncx2 as boost_ncx2,
+)
+
+
+def test_issue_11026():
+    p = 0.5
+    assert boost_bernoulli(p).median() == 0
+    assert boost_bernoulli(p).interval(1) == scipy_bernoulli(p).interval(1)
+    assert boost_bernoulli(p).interval(0.999) == scipy_bernoulli(p).interval(0.999)
+    assert boost_bernoulli(p).interval(1-1e-14) == scipy_bernoulli(p).interval(1-1e-14)
+
+def test_issue_12635():
+    p = 0.9999999999997369
+    a, b = 76.0, 66334470.0
+    tol = 1e-2
+    assert not scipy_beta.ppf(p, a-1, b) - scipy_beta.ppf(p, a, b) < tol
+    assert boost_beta.ppf(p, a-1, b) - boost_beta.ppf(p, a, b) < tol
```

Comment:
```suggestion
    # Confirm that Boost's beta distribution resolves gh-12635. Check against R:
    # options(digits=16)
    # p = 0.9999999999997369
    # a = 75.0
    # b = 66334470.0
    # print(qbeta(p, a, b))
    assert_allclose(boost_beta.ppf(p, a, b), 2.343620802982393e-06)
```
I don't think we need to test that SciPy's distribution does the wrong thing. Let's just add a direct test that Boost's `beta` does the right thing.

Reply:
On second thought with these... I think for the actual PR we may want to _actually_ replace SciPy stats distributions with the Boost equivalents. In that case, these tests I'm writing should go in `test_distributions.py`.

Reply:
No files moved; scipy distributions removed; tested against R output